etl_i2b2/sql_scripts/cms_enr_dstats.sql

```diff
@@ -101,23 +101,23 @@ where buyin != '0' -- Not entitled
   select * from cms_deid_12caid_16care.mbsf_abcd_summary
 )
 , per_bene_mo_456 as (
-select bene_id, bene_enrollmt_ref_yr, mo, buyin, extract_dt
+select bene_id, bene_enrollmt_ref_yr, mo, buyin, hmo, extract_dt
 from per_bene_456s
 unpivot(
-        buyin
+        (buyin,hmo) /*add hmo column*/
```

Comment:
again, `/*add hmo column*/` seems more like change history than documentation of the current design.

Reply:
space added and comment removed in new commit